src/main/daml/Daml/Finance/Interface/Types/Fpml.daml

```diff
@@ -24,6 +26,62 @@ data CalculationPeriodDates = CalculationPeriodDates
     calculationPeriodFrequency : CalculationPeriodFrequency
   deriving (Eq, Show)
 
+data ResetDates = ResetDates
```

Comment:
For my understanding, what is the difference between reset dates and fixing dates?